ci-benchmark/benchmark.graphql

```diff
@@ -1,8 +1,10 @@
-schema @server(port: 8000, hostname: "0.0.0.0") @upstream(baseURL: "http://jsonplaceholder.typicode.com") {
+schema
+  @server(port: 8000, hostname: "0.0.0.0", enableJIT: true)
+  @upstream(baseURL: "http://jsonplaceholder.typicode.com", httpCache: 42) {
   query: Query
 }
 
-type Query @cache(maxAge: 30000) {
```

Comment:
.. we were performing benchmarks and results were too good to be true so instead of this I switched to httpCache which is part of IR

will revert this